PartymodeAutostart.py

```diff
@@ -57,6 +62,9 @@ def runPartyMode(self):
 
         self.activateVisualisation()
 
+        if (self.volumeBarPartyMode):
+            xbmc.executebuiltin("XBMC.SetVolume(%d,true)" % self.volumePartyMode)
+
```

Comment:
It was because of problems with my skin, and I figure it might be a problem elsewhere.  I set the volume the first time to make sure it has been set before the music starts, then the second time to make sure the dialog pops up.  Noticed it doesn't work on all skins either, but didn't see another option for fixing it.